### ISSUE TITLE
[BUG] lr scheduler not changed when resume and args is changed

### ISSUE THREAD

**Issue by triwahyuu** [nodefluxio/vortex#70]
**Describe the bug**
When resume training and the scheduler args is changed, the scheduler behavior is not changed.

**To Reproduce**
1. train model with this lr_scheduler config:
```yaml
lr_scheduler: {
    method: StepLR,
    args: {step_size: 200}
}
```
2. resume training with this config:
```yaml
lr_scheduler: {
    method: StepLR,
    args: {step_size: 350}
}
```
3. the learning rate still stepped when hit epoch 200.

**Additional context**
this happened because the lr scheduler load the state dict from the resumed model, should be changed.
